genapi.py

```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
 def setup(ns):
     """Ensure that we are ready to perform code generation. Returns typesystem."""
     # load raw table
-    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
+    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
```

Comment:
is there a reason this has to be in the src_dir? why not hold the .js file and write the json file to the build dir? that would keep things "mostly" the same.